app/views/returns/vatReturnDetails.scala.html

```diff
@@ -116,7 +116,7 @@ <h3 class="column-two-thirds bold-small section">@messages("yourVatReturn.vatDet
     </div>
     <p id="adjustments">@messages("yourVatReturn.adjustments")</p>
 
-    @if((appConfig.features.payments()) && (vatReturnViewModel.moneyOwed && !vatReturnViewModel.isRepayment)) {
+    @if((appConfig.features.allowPayments()) && (vatReturnViewModel.moneyOwed && !vatReturnViewModel.isRepayment)) {
```

Comment:
Can you remove the parenthesis around the `appConfig.features.allowPayments()`. They aren't needed